codebasin/finder.py

```diff
@@ -159,11 +161,31 @@ def find(
                         f"{filename} found in definition of platform {p} "
                         + "but missing from codebase",
                     )
-            state.insert_file(e["file"])
+            filenames.add(e["file"])
+    for f in tqdm(
+        filenames,
+        desc="Parsing",
+        unit="file",
+        leave=False,
+        disable=not show_progress,
+    ):
+        state.insert_file(f)
 
     # Process each tree, by associating nodes with platforms
-    for p in configuration:
-        for e in configuration[p]:
+    for p in tqdm(
+        configuration,
+        desc="Preprocessing",
+        unit="platform",
```

Comment:
Space before `platform`

Reply:
Fixed in 4703339b54637b119aaba86ac73b37537aba4505.